src/Helper/CkoLogger.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Checkoutcom\Helper;
+
+use Checkoutcom\Config\Config;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Client;
+use RuntimeException;
+use Checkoutcom\Helper\Utilities;
+use Psr\Log\LoggerInterface;
+use Monolog\Formatter\JsonFormatter;
+use Checkoutcom\Handler\DatadogHandler;
+
+
+/**
+ * This class is used to log on clouEvent and shopware 6 without
+ * throwing an exception
+ * 
+ */
+class CkoLogger {
+
+    /**
+     * @var LoggerInterface
+     */
+    public static $logger;
+
+    function __construct(LoggerInterface $logger) {
```

Comment:
In the constructor we need to initialize logger (which needs to be done once)
and then in log, we only proxy to logger itself.

Reply:
For example:

```
class CkoLogger {

    public static $logger;

    function __construct(LoggerInterface $logger) {
        self::$logger = $logger;

        if (config::logcloudEvent() == true) {
            self::$logger->pushHandler(new DatadogHandler(
                Url::getCloudEventUrl(), $logLevel, true
            ));
        }
    }

    public static function log($message, $scope, $type, $id = false, $logLevel) {
        self::$logger->$logLevel($body);
    }
}
```